Configuration/FlexForms/flexform_div.xml

```diff
@@ -15,8 +15,8 @@
 							<label>LLL:EXT:theme_t3kit/Resources/Private/Language/ContentElements.xlf:iconTextButton.flexform.iconClass</label>
 							<config>
 								<type>user</type>
-								<userFunc>T3kit\T3kitExtensionTools\UserFunction\IconFontSelector->renderField</userFunc>
-								<cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/fonts/style.css</cssFile>
+								<userFunc>T3kit\themeT3kit\UserFunction\IconFontSelector->renderField</userFunc>
+								<cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/dev/fonts/style.css</cssFile>
```

Comment:
Even if we are not using felayout_t3kit as a separate repo I would like to keep it as a name of the folder inside of theme_t3kit. Just to let everyone know that it is felayout as it was in previous versions, and all FE stuff is here.